spec/models/cracked_ineffective_trial_spec.rb

```diff
@@ -1,35 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe CrackedIneffectiveTrial, type: :model do
-  subject(:cracked_ineffective_trial) { described_class.new(body: cracked_ineffective_trial_hash) }
+  subject(:cracked_ineffective_trial) { described_class.new(body: cracked_ineffective_trial_data) }
 
-  let(:cracked_ineffective_trial_hash) do
-    JSON.parse(file_fixture("cracked_ineffective_trial.json").read)
-  end
-
-  describe "#id" do
-    subject { cracked_ineffective_trial.id }
-
-    it { is_expected.to eql("c4ca4238-a0b9-3382-8dcc-509a6f75849b") }
-  end
-
-  describe "#type" do
-    subject(:type) { cracked_ineffective_trial.type }
-
-    it "is expected to downcase the type" do
-      expect(type).to eql("cracked")
+  context "with required fields" do
+    let(:cracked_ineffective_trial_data) do
+      JSON.parse(file_fixture("cracked_ineffective_trial/required_fields.json").read)
```

Comment:
So there is an optional field of `data` - do you want to do a context with all fields and another file_fixture? Or is that unnecessary/overkill?